src/reporters/json.ts

```diff
@@ -93,8 +198,11 @@ export default class JSONReporter extends BaseReporter {
       }) => {
         if (networkinfo) {
           networkinfo.forEach(ni => {
-            this.writeJSON('journey/network_info', journey, {
+            this.writeJSON({
+              type: 'journey/network_info',
+              journey,
               timestamp: ni.timestamp,
+              root_fields: formatNetworkFields(ni),
               step: ni.step,
               payload: snakeCaseKeys(ni),
```

Comment:
So, it looks like we're writing the ECS fields in addition to the raw network data. Seems like we could possibly just drop payload with this patch?

Reply:
Discussed offline, we will add all payload network fields for now to root_fields and drop payload on next version.